nuxeo-features/nuxeo-platform-audit/nuxeo-platform-audit-core/src/test/java/org/nuxeo/ecm/platform/audit/TestAuditStorage.java

```diff
@@ -46,5 +51,6 @@ public void testStartsWith() throws Exception {
     protected void flush() throws Exception {
         TransactionHelper.commitOrRollbackTransaction();
         TransactionHelper.startTransaction();
+        Assert.assertTrue(Framework.getService(AuditLogger.class).await(5, TimeUnit.SECONDS));
```

Comment:
Use static import for `assertTrue` please.
`AuditLogger` can be injected instead of an explicit `Framework.getService` lookup